corehq/apps/reports/tasks.py

```diff
@@ -46,24 +47,25 @@
 logging = get_task_logger(__name__)
 EXPIRE_TIME = ONE_DAY
 
+_calc_props_soft_assert = soft_assert(to='{}@{}'.format('dmore', 'dimagi.com'))
```

Comment:
should this have `use_exponential_backoff` set to `False`?

Reply:
For some reason I thought that was the default, but I'm not sure why. https://github.com/dimagi/commcare-hq/pull/25591/commits/3d9c797d373709058e7e56712b9a9efc7a6568c9 :eagle: :eye: